src/components/Board.js

```diff
@@ -5,29 +5,84 @@ import axios from 'axios';
 import './Board.css';
 import Card from './Card';
 import NewCardForm from './NewCardForm';
-import CARD_DATA from '../data/card-data.json';
+// import CARD_DATA from '../data/card-data.json';
 
 class Board extends Component {
-  constructor() {
-    super();
-
+  constructor(props) {
+    super(props);
+    const {url, boardName} = this.props;
     this.state = {
       cards: [],
+      error: [],
+      baseUrl: url,
+      boardName: boardName,
+      deleteCardUrl: "https://inspiration-board.herokuapp.com/cards/"
     };
   }
 
+  componentDidMount(){
+    this.getCards();
+  }
+
+  getCards = () => {
+    axios.get(this.state.baseUrl + this.state.boardName + "/cards")
+    .then((response) => {
+      this.setState({ cards: response.data });
+    })
+    .catch((error) => {
+      this.setState({ error: error.message });
+    });
+  }
+
+  deleteCardCallback = (cardID) => {
+    axios.delete(this.state.deleteCardUrl + cardID)
+    .then((response) => {
```

Comment:
I like that you've kept all the API interaction logic in one component - the callbacks are a little more complex, but I would say it makes the app as a whole much easier to comprehend. Whether or not you intended it, this is a great example of the container component pattern well-applied.